src/reports/tables/Aggregate5.jsx

```diff
@@ -1,53 +1,65 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React from "react";
+import PropTypes from "prop-types";
 
 const renderData = applicantIncomes => {
   return applicantIncomes.map((applicantIncome, index) => {
     return [
       renderApplicantIncome(applicantIncome.applicantIncome, index),
       renderCharacteristics(applicantIncome.borrowerCharacteristics)
-    ]
-  })
-}
+    ];
+  });
+};
 
 const renderApplicantIncome = (applicantIncome, index) => {
   return (
     <tr key={index}>
       <th
         colSpan={13}
         style={{
-          borderTopWidth: '2px',
-          fontWeight: 'bold',
-          textTransform: 'uppercase',
-          backgroundColor: '#f1f1f1'
+          borderTopWidth: "2px",
+          fontWeight: "bold",
+          textTransform: "uppercase",
+          backgroundColor: "#f1f1f1"
         }}
       >
         {applicantIncome}
       </th>
     </tr>
-  )
-}
+  );
+};
+
+const convertObjToArray = obj => {
+  return Object.entries(obj).map(([key, value]) => ({ [key]: value }));
```

Comment:
Object.entries is unsupported in ie and so this function will need to be rewritten